crates/manifest/src/schema/common.rs

```diff
@@ -30,13 +30,21 @@ pub enum ComponentSource {
         /// `digest = `"sha256:abc123..."`
         digest: String,
     },
+    /// `{ ... }`
+    Registry {
+        /// `package = "example:component"
+        package: String,
+        /// `version = "1.2.3"
+        version: String,
```

Comment:
Any reason to not use `semver::Version` here or are we keeping it fairly generic at this point in loading?

Reply:
Technically "not realising that `wasm_pkg_loader::Version` _is_ `semver::Version` is a reason so yes